<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionUpdateQuest</class>
 <widget class="QWidget" name="ActionUpdateQuest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblQuest">
       <property name="text">
        <string>Quest:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblPhase">
       <property name="text">
        <string>Phase:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="phase">
       <property name="minimum">
        <number>-2</number>
       </property>
       <property name="maximum">
        <number>127</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblForceUpdate">
       <property name="text">
        <string>Force Update:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="forceUpdate"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblFlagSetMode">
       <property name="text">
        <string>Flag Set Mode:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="flagSetMode">
       <item>
        <property name="text">
         <string>Update</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Increment</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Decrement</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblFlagStates">
       <property name="text">
        <string>Flag States:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="ActionMap" name="flagStates" native="true"/>
     </item>
     <item row="0" column="1">
      <widget class="ObjectSelector" name="quest" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActionMap</class>
   <extends>QWidget</extends>
   <header>ActionMap.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ObjectSelector</class>
   <extends>QWidget</extends>
   <header>ObjectSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
